backend/guides/backend_api/hierarchical_connection.md

```diff
@@ -0,0 +1,37 @@
+# Hierarchical Connection Between Different Layers Of The Backend:
+
+## Current Layout:
+
+```
+             |------------ Mappers --------------------|
```

Comment:
Would it be better to make the Controllers point to Mappers, Services, and Middlewares? On line 28, it mentions that Controllers depend on Mappers, Services, and Middlewares, yet Controllers isn't pointing to either of them.

If so, would the diagram look like this instead?

```
             |-----------> Mappers --------------------|
             |                 |                       |
Controllers--|                \|/                      |
             |-----------> Services ------------ Repositories ------------ Database
             |                /|\
             |                 |
             |-----------> Middlewares
```

Reply:
I specified on line 25 of the file that I only use the arrows for layers that are on the same level, just so readers know how layers on the same level depend on each other

Reply:
I see. Makes sense now, thank you!